packages/styleUtils/typography/stories/textBlock.stories.tsx

```diff
@@ -47,9 +48,11 @@ export const Default = () => (
   </Container>
 );
 
-export const UsingTypographyComponents = () => (
+export const Default = Template.bind({});
+
+export const UsingTypographyComponents = args => (
   <Container>
-    <TextBlock>
+    <TextBlock {...args}>
       <div>
         <HeadingText1>Lorem Ipsum</HeadingText1>
```

Comment:
It might be useful to say which typography components are being used within this the TextBlock story, in case the user isn't aware of what's considered a Typography components.

Reply:
This is a good idea! I will add this to improve the story using UI Kit components. Users can also see every component used when they click the "Show Code" on the Docs page to see how each story was written.